format-blob/pom.xml

```diff
@@ -61,6 +61,10 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+      <dependency>
```

Comment:
this has indentation issues.

Also, why is this dependency being added? Which class from hydrator-test is needed?

Reply:
Removed